src/source/RazorEngine.Core/Compilation/CompilerServiceBuilder.cs

```diff
@@ -51,9 +51,9 @@ public static ICompilerService GetDefaultCompilerService()
         {
             var config = RazorEngineConfigurationSection.GetConfiguration();
             if (config == null)
-                return GetCompilerService(Language.CSharp);
+                return GetCompilerService(Language.CSharp, null);
 
-            return GetCompilerService(config.DefaultLanguage);
+            return GetCompilerService(config.DefaultLanguage, null);
```

Comment:
The point of this method is to generate a default CompilerService with a language specified - either CSharp or what's defined in the config.  The default values in the passed parameter should be null specifically because this is default (plus `config` isn't the same type as `ITemplateServiceConfiguration`)